app/build.gradle

```diff
@@ -214,6 +214,10 @@ dependencies {
     implementation 'androidx.recyclerview:recyclerview:1.1.0'
     implementation "androidx.room:room-runtime:${androidxRoomVersion}"
     implementation "androidx.room:room-rxjava3:${androidxRoomVersion}"
+
+    // Apple Silicon Mac workaround (https://issuetracker.google.com/issues/174695268#comment9)
+    kapt "org.xerial:sqlite-jdbc:3.34.0"
+
```

Comment:
Please remove those changes; they are unrelated to this PR.

Reply:
Sorry, that's needed for me to build the code. Is there any way I can revert them in this PR? I'll avoid committing this next time.

Reply:
Just remove the lines manually and make a commit. We squash PRs here when merging them anyway.

Reply:
Oh, I see. Will do.

Reply:
Done in https://github.com/TeamNewPipe/NewPipe/pull/6965/commits/4f5e7721576faebf0b338152f1600cfb307d02da

Reply:
> We squash PRs here when merging them anyway.

Well, usually not, if (almost) every commit is meaningful. I only squash PRs in cases like this. But it's ok ;-)